microsoft-edge/webdriver-chromium/index.md

```diff
@@ -35,24 +37,32 @@ The functional relationship between these components is as follows:
 | Technology | Role |
 |---|---|
 | WebDriver | A W3C standard for a platform- and language-neutral wire protocol.  This protocol allows out-of-process programs to remotely instruct the behavior of web browsers. |
-| Microsoft Edge WebDriver | Microsoft's implementation of the WebDriver protocol specifically for Microsoft Edge.  Test authors write tests that use WebDriver commands that Microsoft Edge WebDriver receives.  Edge WebDriver is then responsible for communicating that command to the browser. |
-| A WebDriver testing framework | Test authors use a testing framework to write end-to-end tests and automate browsers.  Provides a language-specific interface that translates your code into commands that are sent to Edge WebDriver.  WebDriver testing frameworks exist for all major platforms and languages.  One such framework is Selenium. |
+| Microsoft Edge WebDriver | Microsoft's implementation of the WebDriver protocol specifically for Microsoft Edge.  Test authors write tests that use WebDriver commands that Microsoft Edge WebDriver receives.  Microsoft Edge WebDriver is then responsible for communicating that command to the browser. |
+| A WebDriver testing framework | Test authors use a testing framework to write end-to-end tests and automate browsers.  Provides a language-specific interface that translates your code into commands that are sent to Microsoft Edge WebDriver.  WebDriver testing frameworks exist for all major platforms and languages.  One such framework is Selenium WebDriver. |
 | Internet Explorer Driver | An open-source implementation of the WebDriver protocol specifically for Internet Explorer.  To run legacy end-to-end tests for Internet Explorer Mode, we recommend using Internet Explorer Driver. |
 | Microsoft WebDriver (legacy) | The previous, browser-specific driver for Microsoft Edge (EdgeHTML), which is also known as Microsoft Edge Legacy. |
 
 The following sections describe how to get started with WebDriver for Microsoft Edge.
 
 
+<!-- ====================================================================== -->
+## Install both a browser driver and a WebDriver framework
+
+You must install both a browser driver (Microsoft Edge WebDriver), and a WebDriver framework (such as Selenium WebDriver).  These are separate components.
```

Comment:
This section can potentially be removed, and the content placed elsewhere.

I think this first sentence would be good to move to the "Download Microsoft Edge WebDriver" section below. It provides some useful context that was missing.

The following paragraph about the Visual Studio template seem like a better fit for the troubleshooting section below since it is specific to a Visual Studio template that we don't see much usage for.

Reply:
done locally